src/components/LandingInfo.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import './LandingInfo.scss';
+import { IonButton } from '@ionic/react';
+
+const LandingInfo: React.FC = () => {
+    return (
+        <div>
```

Comment:
Just waiting on this to be changed to Ionic components and we'll go ahead and merge